### PR TITLE
HOCS-3625: update call for all teams by shortcode to exclude deactiva…

### PR DESCRIPTION
**Pull request by gary-newbolt**
…ted and add test

### CODE REVIEW

File: src/test/java/uk/gov/digital/ho/hocs/info/api/TeamIntegrationTests.java [hunk 1]
```diff
@@ -311,6 +311,36 @@ public void shouldReturnPreConditionFailedErrorWhenTryingToDeleteTeamWhichHasAct
...
+
+        assertThat(teamDtos.size()).isEqualTo(2);
+        assertThat(teamDtos.stream().map(TeamDto::getDisplayName)).noneMatch(el -> el.equals("TEAM_103")
+                        || el.equals("TEAM_104")
+                        || el.equals("TEAM_102")
+        );
+        assertThat(teamDtos.stream().map(TeamDto::getDisplayName)).allMatch(el -> el.equals("TEAM_101")
+                || el.equals("TEAM_100")
+        );
+
```
Comment (LaurieBennett) [inline src/test/java/uk/gov/digital/ho/hocs/info/api/TeamIntegrationTests.java:336]: Can we remove these empty lines?

**Review by LaurieBennett — CHANGES_REQUESTED** [review]
Remove some empty lines and this can pass

File: src/test/java/uk/gov/digital/ho/hocs/info/api/TeamIntegrationTests.java (hunk 1 above)
Comment (gary-newbolt) [reply src/test/java/uk/gov/digital/ho/hocs/info/api/TeamIntegrationTests.java]: Just pushed a change to remove. Thanks

**Review by LaurieBennett — APPROVED** [review]
Code looks good, pass

**Pull request merged**